packages/pyright-internal/src/tests/languageServer.test.ts

```diff
@@ -235,4 +236,301 @@ describe(`Basic language server tests`, () => {
         // Make sure the error has a special rule
         assert.equal(diagnostic.diagnostics[0].code, 'reportUnknownParameterType');
     });
+    describe('module renaming', () => {
```

Comment:
add tests for `from . import y` when `y` -> `z`, wither that it works, or that it doesn't do anything

Reply:
8fc06590f9629f164c0afb5092bf2cf2766f15d1